### ISSUE TITLE
where is the GPT-Net?

### ISSUE THREAD

**Issue by flyingtiger111** [areylng/SLSG#1]
请问 文章中 提到的 GPT-Net 和 80% mask 对应在代码的哪一部分？我没有找到。请作者指明

**Comment by HU13-SVG** [comment]
class Encoder(nn.Module):
    def __init__(self, pretrained=True, head_layers=[512,512,512,512,512,512,512,512,128], num_classes=2,
                 data_type=None,use_patch_core=False,encoder_freeze=False,epoch_MODE=0):
        super(Encoder, self).__init__()
        self.epoch_MODE=epoch_MODE
        self.resnet18 = resnet18(pretrained=True)
        try:
            weights = torch.load(f"model_data/pre_models/{data_type}.pth")  # 75  better_E_final
            self.resnet18.load_state_dict(weights)
        except:
            pass
        self.Init()
我也没有找到GPT-Net，但是这里提供了一个线索。显然作者没有提供model_data/pre_models这个参数，还是使用的默认的resnet18的参数
